<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matrix_zuochu</class>
 <widget class="QWidget" name="Matrix_zuochu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matrix_zuochu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTextEdit" name="Edit_show_of_result"/>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_read_txt_Data">
        <property name="text">
         <string>读取Matlab数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_zuochu">
        <property name="text">
         <string>预留</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_get_matrix">
        <property name="text">
         <string>左除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
